libcxx/test/std/containers/container.adaptors/flat.map/flat.map.access/at_transparent.pass.cpp

```diff
@@ -106,6 +106,13 @@ int main(int, char**) {
     TEST_IGNORE_NODISCARD m.at(Transparent<int>{3});
     assert(transparent_used);
   }
+  {
+    // std::string and C string literal
```

Comment:
Can you please add the LWG issue number as a comment here? And to the other test cases.